### ISSUE TITLE
draft box checked, "publish" showed up instead of "save"

### ISSUE THREAD

**Issue by steviepubliclab** [publiclab/plots2#6123]
Hello! 
I haven't been able to recreate this issue, so I'm not sure what the bug really is, but I was trying to save a draft research note to come back to later, I had the draft box checked, but instead of seeing "save" I saw "Publish." Clicking and unclicking "save as draft" didn't change the button. Then, just to test it out, I clicked "publish" and even though the "save as draft" button was checked, it published my Research Note.

I deleted that note, and tried to recreate the bug but was unable to do so. No worries, on dealing with this, just wanted to share it out in case anyone else runs into this!

**Comment by newbazz** [comment]
can you please write steps to reproduce the issue, tbh reading the whole paragraph to understand the issue is tiresome :P

**Comment by gauravano** [comment]
Will try to reproduce @steviepubliclab. Thanks for reporting!

**Comment by gauravano** [comment]
One case where I saw this bug is -- when I gone back back after saving my note as draft. At that time, title was filled and so is save as draft checkbox selected, but button was "publish". Hmm, we have to reset the checkbox after publish is clicked. Or, we can set the button according to checkbox status rather than changing button on checkbox click.

**Comment by steviepubliclab** [comment]
Oh interesting! I don't think I had successfully saved my first version as a draft, but I might have hit the "back" button because I remember clicking onto a link by accident and needing to go back to the Research Note.

**Comment by gauravano** [comment]
Yeah, on page load, the publish is loaded by default and it's changes to save when triggered via checkbox. On going back, page is loaded but not JS, and so they are un-synced. Putting condition on button using checkbox state will solve this.

**Comment by steviepubliclab** [comment]
wow!! I can't believe you found this!
